adb_graphics/default_specs.yml

```diff
@@ -609,6 +621,50 @@ ptmp: # Potential temperature
     ticks: 10
     unit: K
     wind: 10m
+ptyp: # Hourly total precipitation
+  sfc:
+    clevs: [0.002, 0.01, 0.05, 0.1, 0.25, 0.50, 0.75, 1.0, 2.0]
+    cmap: gist_ncar
+    colors: pcp_colors
+    contours:
+      csnow_sfc:
+        colors: blue
+        levels: [0, 1]
+        linewidths: 0.2
+      crain_sfc:
+        colors: green
+        levels: [0, 1]
+        linewidths: 0.2
+      cfrzr_sfc:
+        colors: purple
+        levels: [0, 1]
+        linewidths: 0.5
+      cicep_sfc:
+        colors: red
+        levels: [0, 1]
+        linewidths: 0.5
+    hatches:
+      crain_sfc:
+        colors: ['', 'green']
+        hatches: ['', '|||', '|||']
+        levels: [0.0, 0.5, 1.0]
+      csnow_sfc:
+        colors: ['', 'blue']
+        hatches: ['', '---', '---']
+        levels: [0, 0.9, 1.1]
+      cfrzr_sfc:
+        colors: ['', 'purple']
+        hatches: ['', '///', '///']
+        levels: [0, 0.5, 1.0]
+      cicep_sfc:
+        colors: ['', 'red']
+        hatches: ['', '\\\\', '\\\\']
+        levels: [0, 0.5, 1.0]
```

Comment:
It might be helpful to arrange these two lists (contours and hatches) in the same order. Now that I'm looking more closely at the code, too, I think we should turn them into actual lists of dicts. Do that by adding a "-" in front of the next heading level under hatches and contours. That _should_ preserve the order of the contours/hatches.

Would you also explore reducing these list to 2 elements? Perhaps the cutoff value and one larger? So, if it's binary, maybe:

```
- cicep_sfc:
  colors: red
  hatches: ['', '\\\\']
  levels: [0.9, 1.1]
```

If this is what it has to be...that's cool, too. This is mostly just my curiosity.